lib/auth0_ex/management/user.ex

```diff
@@ -10,7 +10,7 @@ defmodule Auth0Ex.Management.User do
   Gets all the users
 
       iex> Auth0Ex.Management.User.all()
-      iex> Auth0Ex.Management.User.all(fields: "name", q: "app_metadata.admin:'true'")
+      iex> Auth0Ex.Management.User.all(%{fields: "name", q: "app_metadata.admin:'true'"})
```

Comment:
@sroze I believe the keyword list way is also gonna work because of the following line: https://github.com/techgaun/auth0_ex/blob/601ade578b2e7f03eb3d3fc1d184e0a6f9b4a023/lib/auth0_ex/api.ex#L27
So, can you still preserve the existing examples?

Reply:
Interestingly, it doesn't work when I use it 🤔

Reply:
@sroze hm. I think we're using keyword list way at our work but I will double check.

Reply:
@sroze just tested, its working fine for me. Whats the error you get? I appreciate you taking time to make sure the documentation is correct.